beesdoo_easy_my_coop/tests/test_res_partner.py

```diff
@@ -162,6 +162,9 @@ def test_compute_can_shop_share_a(self):
         Test that a cooperator can shop based on his share type.
         """
         coop1 = self.env.ref("beesdoo_base.res_partner_cooperator_1_demo")
+        coop1.cooperative_status_ids = self.env.ref(
+            "beesdoo_shift.cooperative_status_1_demo"
+        )
```

Comment:
```suggestion
         coop1 = self.env.ref("beesdoo_base.res_partner_worker_1_demo")
```

You can also attempt this.

Reply:
Yes I tried this solution but it had complication because the `is_worker` field was overiden by beesdoo_easy_my_coop, so I needed to replace the attribute value.

Reply:
Ah, awesome. Can you add a fixup commit with a comment of what you just wrote? Some day it may be useful, especially if we refactor out the demo partner data from beesdoo_shift as we probably should.

Reply:
You mean an empty fixup commit with just the comment ?

Reply:
done :)